package.json

```diff
@@ -4,29 +4,32 @@
   "devDependencies": {
     "@types/jasmine": "^3.5.10",
     "clang-format": "^1.2.2",
+    "generate-license-file": "^1.2.0",
     "husky": "^1.3.1",
     "jasmine": "^3.5.0",
+    "prettier": "^2.4.1",
+    "pretty-quick": "^3.1.1",
     "tslint": "^5.9.1",
     "typescript": "^4"
   },
   "engines": {
-    "yarn": ">=1.0",
-    "node": "^12"
+    "node": ">=16.13.0",
+    "npm": ">=8.1.0"
   },
   "scripts": {
+    "action": "bash ./scripts/run_action.sh",
+    "action:list": "find . -name '*.action.sh' | sed -E 's:./src/(.*).action.sh:\\1:' | grep -v 'scripts/run_action'",
```

Comment:
I think it would be more ergonomic for this to be the default behavior of `npm run action` with no arguments.

Reply:
I like that. The logic could live inside the `run_action.sh` file, so we don't need to leak those details.

Reply:
Me too. Coming right up!